README.md

```diff
@@ -52,7 +52,19 @@ The Docker extension contributes explorers to view and manage containers, images
 
 ![hideView](resources/readme/hideView.png) ![hideExplorer](resources/readme/hideExplorer.png)
 
-#### Registries
+### Debugging in Containers
+
+With the Docker extension installed, you can easily debug .NET Core and Node.js apps within a container. Debugging support is enabled automatically when you dockerize your code with `Docker: Add Docker Files to Workspace`. To enable debugging for code that's already dockerized, press `F1` and do `Docker: Initialize for Docker Debugging`.
```

Comment:
Instead of being explicit about use of `F1`, mention the command palette (as that's not the shortcut used by everyone).

Reply:
To be fair, I've noticed other extensions using the <kbd>F1</kbd> key, so maybe it's just a thing.  That said, I've never used that shortcut to get to the command palette (on my Mac), usually opting for <kbd>⌘</kbd>+<kbd>P</kbd>.

Reply:
Yeah, I typically use Ctrl + Shift + P (Windows/Linux) myself; old habit from Sublime Text. I'll change the text to make it say something like "Open the command palette (`F1` by default)"